changelog.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+Release 0.5.9
```

Comment:
compatibility-breaking changes should increase major version too. I think it will `0.6` after all

Reply:
@renskiy done.